### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VeliML**
Mitsurunner were originally written so that anybody from lampopumput.info could read and have a glue what is happening in script. Just to prevent people to fear the system. Now there is already users using it on real hardware proofing that this works. Thus this is not anymore needed to be written in Finnish language

### CODE REVIEW

File: mitsurunner.yaml [hunk 1]
```diff
@@ -1,193 +1,173 @@
...
-# Globaalit muuttujat, joilla estetään esim. alle minimiajan tapahtuvat sulatukset.
+# global varialbes and their initial values
 globals:
-  - id: G_minimiaika_kulunut
+  - id: G_min_heating_time_passed #variable controlling minimum heating time after defrosting
     type: bool
     restore_value: no
     initial_value: 'true'
-  - id: G_ylityksen_minimiaika_kulunut
+  - id: G_excess_min_time_passed #variable preventing defrosting untill excess time has been passed
```
Comment (kestopuu) [inline mitsurunner.yaml:21]: typo: untill -> until

File: mitsurunner.yaml [hunk 2]
```diff
@@ -1,193 +1,173 @@
...
-# tee kytkentä ja anna laitteen tunnistaa dallasit. Tarkista logeista, mitkä osoitteet sinne tulostuu. 
-# lämmitä kädellä toista sensoria ja merkitse se. elä sekoita enää sensoreita tämän jälkeen
-# valitse, kummasta tulee kennon lämpöä mittaava sensori ja kummasta ulkolämpötä mittaava
-# heitin nyt tarkkuudeksi kaksi desimaalia. On ainakin tarkka...
+    pin: $relay_pin
+    name: "Defrost prevention control"
+    id: gpio_relay
+
+# Dallas DS18B20 temperature sensor details. 
+# you need to check IDs of temperature sensors by starting this SW and checking those from logs printing to termina.
```
Comment (kestopuu) [inline mitsurunner.yaml:43]: typo: termina -> terminal

File: mitsurunner.yaml [hunk 3]
```diff
@@ -1,193 +1,173 @@
...
+# you need to check IDs of temperature sensors by starting this SW and checking those from logs printing to termina. 
+# After checked, copy IDs to platform.yaml file and heat another sensor. Check which sensor it were and use it for the appropriate topic
 sensor:
   - platform: dallas
-    address: $dallas_address_kenno # tämä on sinulla eri aina ja jokaiselle sensorille uniikki. Näet tämän logeista kun kommentoit tämän rivin pois
-    name: "kennonlampotila"
-    id: kennonlampo
+    address: $dallas_address_heat_exhanger_temp # Please remember to define this on platform.yaml
+    name: "outdoor unit heat exhanger temperature"
+    id: heat_exhanger_temp
```
Comment (kestopuu) [inline mitsurunner.yaml:49]: Taitaa olla exchanger oikea termi, eli c väliin.

File: mitsurunner.yaml [hunk 4]
```diff
@@ -1,193 +1,173 @@
...
+                id(trigger_defrosting).execute();
+                reset_min_time_of_temperature_delta = true;
                 }
-            // JOS ylityksen minimiaikatimer on käynnisä, odotellaan vain... Eli tyhjä blokki
-            else if ( id(kaynnista_lampotileron_minimi_aika).is_running() ) {
+            // if excess timer were not yet passed, do nothing
+            else if ( id(start_excess_min_time_timer).is_running() ) {
                 }
-            // Ei ylityksen minimiaikaa käynnissä. Käynnistä se, koska  muut ehdot täyttyvät lämmitykselle. 
+            // excess min itme were not on, but other defrosting requirements were fullfilled to start it
```
Comment (kestopuu) [inline mitsurunner.yaml:95]: typo: itme -> time

File: platform.yaml [hunk 5]
```diff
@@ -1,55 +1,50 @@
...
   broker: 192.168.1.xxx 
   id: mqtt_client
-#kirjoita brokerillasi tunnistettava username alapuoliseen riviin : merkin jälkeen (yksi välilyönti ennen käyttäjänimeä)
-  username: vaihda_tämä
-#client_id tarvitaan ainakn IoT Guruun:
-  client_id: vaihda_tämä
-#kirjoita brokerisi käyttäjän salasana alapuoliseen riviin : merkin jälkeen (yksi välilyönti ennen käyttäjänimeä)
-  password: vaihda_tämä
-#mqtt viestin alkuosa, muuta tämä haluamaksesi tai pidä tällaisena
+  username: "change_this"
```
Comment (kestopuu) [inline platform.yaml:27]: Nämä käsittääkseni pitäisi olla ilman lainausmerkkejä, joten laittaisin nuo ghancge_this myös ilman lainausmerkkejä selkeyden vuoksi. Lähde:
https://esphome.io/components/mqtt.html

File: platform.yaml [hunk 6]
```diff
@@ -1,55 +1,50 @@
...
+
+# addresses for your dallas DS18B20 temperature sensors - you need to update these
+  dallas_address_heat_exhanger_temp: '0x653C01D607697928'
+  dallas_address_outdoor_temp:       '0x93000802B67D1E10'
+  
+  topic_heatexhanger:      'pub/xxx/xxx/xxx/lampotila' #if you will use IOT guru, please uncomment and update this to get graphs working
+  topic_outdoor:           'pub/xxx/xxx/xxx/lampotila' #if you will use IOT guru, please uncomment and update this to get graphs working
+# IO-pins:
+  dallas_pin:         'D2' # pin for DS18B20 1-wire temperature sensors
+  relay_pin:           'D0' # pin for controlling relay
```
Comment (kestopuu) [inline platform.yaml:50]: Rivinvaihto loppuun niin git on tyytyväinen ;)

**Review by kestopuu — APPROVED** [review]

**Review by VeliML — COMMENTED** [review]
Find and Replace translation has produced several typos. 
I will correct those and put in with different commit to be able to verify compilation after changes + functionality.

**Pull request merged**